spec/src/run.rs

```diff
@@ -520,55 +425,34 @@ pub fn spec(name: &str) {
                     }
                 }
             }
-            &test::Command::AssertInvalid {
-                line, ref filename, ..
-            } |
-            &test::Command::AssertMalformed {
-                line, ref filename, ..
-            } |
-            &test::Command::AssertUnlinkable {
-                line, ref filename, ..
-            } => {
-                let module_load = try_load(tmpdir.as_ref(), filename, &mut spec_driver);
+            CommandKind::AssertInvalid { module, .. }
+            | CommandKind::AssertMalformed { module, .. }
```

Comment:
isn't `|` better at the end of the lines?

Reply:
I think this depends on the taste. 
As this is an automatic formatting I rather leave it as is.